bot/commands.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func (b *Bot) handlePrivateMessage(message twitch.PrivateMessage) {
-	if message.User.Name == b.cfg.Admin {
+	if contains(b.cfg.Admins, message.User.Name) {
```

Comment:
just wanted to say, this does a linear scan on all admins on every single PRIVMSG message received, could be a performance bottleneck with many admins
the right data structure would be an unordered set :)

Reply:
Then, how about changing it, so message content will be scanned and compared to a "prefix" that bot uses for all its commands and only continue with other checks if the message content starts with `!justlog`?